cwr/testdata/testfile.cwr

```diff
@@ -0,0 +1,2574 @@
+HDRSO000000052PRS FOR MUSIC                                01.102016070119333420160701
+GRHACK0000102.100000000000
+ACK0000000000000000201606301156440000100000000NWRALIVE AND LIVING NOW                                        256599              250310HN            20160630AS
+MSG0000000000000001F00000000NWRF005COPYRIGHT DATE WAS NOT A VALID DATE - REPLACED WITH ZEROS
+NWR0000000000000002ALIVE AND LIVING NOW                                        EN256599        T911053629300000000            POP000000YMTX   ORI         IWONA WYRZYKOWSKA             WYRZYKOWSK  N00000000000
+SPU00000000000000030101283329 NIGHT GARDEN MUSIC                            E 00000000000130046332              021016660210333302103333 N
+SPU00000000000000040100020412 UNIVERSAL MUSIC PUBLISHING N.V.               SE00000000000418430181              055000000550000005500000 N                            379021

```

Comment:
We should use a CWR with completely made up data.

Reply:
Why ?